components/basecomponents/Story/storyStore.ts

```diff
@@ -18,12 +11,26 @@ export interface StoryPhrase {
   end_uk: number;
 }
 
-export const STORIES: Record<string, StoryPhrase[]> = {
-  'pernikova-chaloupka': oPernikoveChaloupce,
-  'dvanact-mesicku': oDvanactiMesickach,
-  'cervena-karkulka': oCerveneKarkulce,
-  'velika-repa': oVelikeRepe,
-  kolobok: oKoblizkovi,
-  'husy-lebedi': oHusach,
-  'ivasik-telesik': oIvasikovi,
+const STORY_SLUGS = [
+  'pernikova-chaloupka',
+  'dvanact-mesicku',
+  'cervena-karkulka',
+  'velika-repa',
+  'kolobok',
+  'husy-lebedi',
+  'ivasik-telesik',
+];
+
+export const getStoryData = async (language: Language, storySlug: string): Promise<StoryPhrase[]> => {
+  if (!STORY_SLUGS.includes(storySlug)) {
+    throw new Error(`Unknown story slug: ${storySlug}`);
+  }
+
+  const storyData = await import(`../../../data/translations/${language}/pohadka_${storySlug}.json`);
+  return storyData.default;
 };
+
+export const STORIES = STORY_SLUGS.reduce((stories, storySlug) => {
```

Comment:
What is the purpose of this data structure? :) 

You are only using it to retrieve individual stories anyway: 
```
storyData = await STORIES[story.slug]((locale as Language) ?? 'cs');
``` 

why not just 

```
 storyData = await getStoryData(locale as Language, story.slug ?? 'cs');
```